spring-xd-starters/spring-xd-module-parent/publish-maven.gradle

```diff
@@ -162,7 +163,7 @@ def customizePom(pom, gradleProject) {
 				dependency {
 					groupId 'org.apache.hadoop'
 					artifactId 'hadoop-common'
-					version '2.3.0'
+					version '2.6.0'
```

Comment:
LGTM; captured @ericbottard 's suggestion in https://jira.spring.io/browse/XD-2694